### ISSUE TITLE
Redundant requests at end of import

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#1666]
While investigating the repro of #1656, @nwt spotted what appeared to be redundant requests coming from Brim.

![](https://user-images.githubusercontent.com/5934157/119019693-49b53000-b952-11eb-8cff-1fc5d792dac2.png)

These were shown to @jameskerr at a team meeting and he agreed to take a closer look.

**Comment by philrz** [comment]
There was an exchange in a PR review (https://github.com/brimdata/brim/pull/1814#issuecomment-914721344) where it was speculated that this issue might have been fixed. I've circled back and checked, and based on what I see in the attached video, it looks like the issue is still with us. Even though the app has moved from the "search" endpoint to the "query" endpoint, there's still redundant requests I can see in DevTools, specifically, two of the `| every 1s count() by _path` and two of the `| head 500` (cc: @jameskerr).

https://user-images.githubusercontent.com/5934157/135920305-c743822c-3775-4902-b7e0-98fe2fe918b0.mp4

**Comment by philrz** [comment]
Verified in Zui commit a9d64be.

While I've not tracked down the precise commit(s) that led to the resolution, the data import workflow has changed significantly in the time since this issue was opened and the problem has gone away during that overhaul. Now the load of data is its own operation, then the **Query Pool** button can be clicked to see the initial splash of the loaded data, and it's at that step that the queries like the ones shown above are issued. Repeating the equivalent repro steps as shown in the attached video, the redundant queries we saw in the past are no longer present.

https://github.com/brimdata/zui/assets/5934157/36c1a619-68c0-4569-abbf-a150b9f161a3

Specifically, the unique queries we see issues here were:

```
"from :pools | id == 0x12b072de308ff796eeb49df7e78c2f5a6646be33 or name == \"0x12b072de308ff796eeb49df7e78c2f5a6646be33\""
"from 'wrccdc.pcap'\n | !has(ts) | count()"
"from 'wrccdc.pcap'\n | ts == null | count()"
"from 'wrccdc.pcap'\n | ts != null | count() by time := bucket(ts, 500ms), group := _path | sort time"
"from 0x12b072de308ff796eeb49df7e78c2f5a6646be33 | min(ts), max(ts)"
"from 'wrccdc.pcap'\n | count()\n  | { i: count(), v: this}\n  | i > 0\n  | head 500\n  | yield v"
"from 'wrccdc.pcap'\n  | { i: count(), v: this}\n  | i > 0\n  | head 500\n  | yield v"
```

These all contribute toward populating the stacked bar chart, the query summary info shown at the bottom of the window, etc.
